### PR TITLE
Add new ANF API 2019-11-01

### PR DESCRIPTION
**Pull request by leonardbf**
Removal of mount target list operation and change to response properties.
Addition of site property.
Removal of snapshot tags property.
Add missing revert volume to snapshot.
Amend property in authorize example.
Other minor amendments.
(Internal ticket ANF-567)

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by leonardbf** [comment]
@iscai-msft you were a suggested reviewer but I also have a question. I'm wondering why the pipeline shows some checks were not successful even though no steps actually show as failed. Do you know why this is? One does show as cancelled. Do you know why this one is cancelled?

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json [hunk 1]
```diff
@@ -0,0 +1,2699 @@
...
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Volumes_Delete": {
+            "$ref": "examples/Volumes_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/capacityPools/{poolName}/volumes/{volumeName}/revert": {
+      "post": {
```
Comment (majastrz) [inline specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json]: >post [](start = 7, length = 4)

you are missing the x-ms-long-running-operation annotation - please take a look at [https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation)

**Review by majastrz — CHANGES_REQUESTED** [review]
Please take a look at the comment I added.

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json (hunk 1 above)
Comment (leonardbf) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json]: Yes. This was also pointed out in the review for the previous swagger which was accepted. We created an open internal ticket on this issue to investigate.

The fact is that adding this results in a failure from the autorest validation 'ERROR (LongRunningResponseStatusCode/R2005/SDKViolation): A 'POST' operation 'Volumes_Revert' with x-ms-long-running-operation extension must have a valid terminal success status code 200 or 201 or 204.'

But perhaps you have some idea how to solve this?

Comment (majastrz) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json]: Yup. Due to limitations of the current version of OpenAPiSpec (aka Swagger), we don't have a good way of modeling the final response on a long running operation (the response on the GET on the async operation URL).

The, somewhat counterintuitive, workaround is to add a 200 response to this path. By convention the code generator will understand that your service is not returning the 200 response for this POST, but that this is the final operation result instead. Does that make sense?

---
In reply to: [369432464](https://github.com/Azure/azure-rest-api-specs/pull/8164#discussion_r369432464) [](ancestors = 369432464)

Comment (leonardbf) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json]: So I just need to add a (dummy) 200 response block. I'll update.

Comment (leonardbf) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json]: @majastrz  Update made. But I notice some validation checks did not run.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by majastrz** [comment]
@NullMDR any idea why some of the checks got cancelled? How do we restart them?

**Comment by leonardbf** [comment]
Any chance this could be merged? I see there is a failure however but it seems to be a problem of the job not the swagger "Failed to run SDK Automation: Error: SDK Automation failed"

**Pull request merged**